components/eam/src/physics/cam/ndrop.F90

```diff
@@ -1632,92 +1609,9 @@ subroutine loadaer_1col( &
 end subroutine loadaer_1col
```

Comment:
There is a comment about "loadaer_1cell" subroutine above, please fix it as we don't have any "loadaer_1cell" routine now.

We can rename `get_aer_num_1col` and `get_aer_mmr_sum_1col` by removing `_1col`.